variables.tf

```
@@ -222,6 +240,12 @@ variable "deregistration_delay" {
   description = "The amount of time to wait in seconds before changing the state of a deregistering target to unused"
 }
 
+variable "connection_termination" {
```

Ah damn, sorry about this, I should have caught it earlier, but can we rename to follow our naming practices?

```suggestion
variable "connection_termination_enabled" {
```

---

:+1: Done in [9ce7192](https://github.com/cloudposse/terraform-aws-nlb/pull/32/commits/9ce7192e3bc0b1c9cf74356ae00c4092f07ebaba)

---

Thanks! Running tests now and should auto-merge when all pass, so check back soon for the new release version! Thanks for the contribution!